js/views/itemEditVw.js

```diff
@@ -217,19 +220,33 @@ module.exports = Backbone.View.extend({
   },
 
   deleteImage: function(e) {
-    var imgIndex = $(e.target).closest('.itemImg').data('index');
+    var imageUploadArray, imgIndex = $(e.target).closest('.itemImg').data('index');
```

Comment:
If you're going to declare variables together, put a return after the , so it's clear they're separate. 

In general, declaring variables near where they are used is deliberate, to make the code more readable for future contributors. While that could create a hoisting issue, we consider that an ok tradeoff.